tools/pipelines/templates/include-publish-npm-package.yml

```diff
@@ -68,10 +75,9 @@ stages:
       buildToolsVersionToInstall: ${{ parameters.buildToolsVersionToInstall }}
       pnpmStorePath: ${{ parameters.pnpmStorePath }}
 
-  # Only generate manifest files for runs in the internal project (i.e. CI runs, not PR run), for the main and release branch, and
-  # for the build of the client release group (we don't need manifests for anything else).
+  # Only generate manifest files for runs in the internal project (i.e. CI runs, not PR run), and for the build of the client release group (we don't need manifests for anything else).
   # Enabling this template for every PR run risks overwriting existing manifest files uploaded to Azure blobs. Therefore, it's crucial to restrict this template to commits merged into the main and release branch.
-  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.tagName, 'client'), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'))) }}:
+  - ${{ if and(eq(variables['System.TeamProject'], 'internal'), eq(parameters.tagName, 'client')) }}:
```

Comment:
I think the old condition is still correct here. If I create a branch named just "myBranch" (not test/myBranch), do we want release reports when I run builds for it? I think the answer is probably no.

Reply:
makes sense. 

curious, where are the bits for "myBranch" published to? Are they still published to the test feed?

Reply:
No, I think those would be published to the build and internal feeds, just like main or the release branches. Only branches named "test/*" will cause the `testBuild` variable to be true so only those get published to the test feed.